muxrpc/test/go/members_test.go

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: 2021 The NGI Pointer Secure-Scuttlebutt Team of 2020/2021
+//
+// SPDX-License-Identifier: MIT
+
+package go_test
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/ssb-ngi-pointer/go-ssb-room/v2/internal/maybemod/keys"
+	"github.com/ssb-ngi-pointer/go-ssb-room/v2/muxrpc/handlers/tunnel/server"
+	"github.com/ssb-ngi-pointer/go-ssb-room/v2/roomdb"
+	"github.com/ssb-ngi-pointer/go-ssb-room/v2/roomsrv"
```

Comment:
```suggestion
	"github.com/ssbc/go-ssb-room/v2/internal/maybemod/keys"
	"github.com/ssbc/go-ssb-room/v2/muxrpc/handlers/tunnel/server"
	"github.com/ssbc/go-ssb-room/v2/roomdb"
	"github.com/ssbc/go-ssb-room/v2/roomsrv"
```

Reply:
`go.mod` lists the import path as `github.com/ssb-ngi-pointer/go-ssb-room/v2`.

Reply:
Oh apologies, I thought that PR was already merged :laughing: Ignore me :running_woman: